src/jit/gentree.cpp

```diff
@@ -11381,11 +11296,11 @@ void Compiler::gtDispArgList(GenTreeCall* call, IndentStack* indentStack)
 // Assumptions:
 //    'tree' must be a GT_LIST node
 
-void Compiler::gtDispTreeList(GenTree* tree, IndentStack* indentStack /* = nullptr */)
+void Compiler::gtDispStmtList(GenTreeStmt* stmts, IndentStack* indentStack /* = nullptr */)
 {
-    for (/*--*/; tree != nullptr; tree = tree->gtNext)
+    for (GenTreeStmt* stmt = stmts; stmt != nullptr; stmt = stmt->gtNextStmt)
     {
-        gtDispTree(tree, indentStack);
+        gtDispTree(stmt->gtStmtExpr, indentStack);
```

Comment:
Shouldn't we display IL offset somewhere?

Reply:
We need a separate function to print statement, probably debug numbers for them, but I was not sure about a name for this function (looks like it should start with gt anymore), so postponed it.

Reply:
These `gt` & co. prefixes are rather horrible. I'd probably just make it `DumpStatement` and call it a day.

Reply:
VS doesn't work well with defines so, in general, these prefixes help to search/grep all occurrences. I do not like any of them, but do not see a nice solution.

Reply:
> VS doesn't work well with defines so, in general, these prefixes help to search/grep all occurrences. I do not like any of them, but do not see a nice solution.

They also hurt writing & reading code (there's something truly horrible about having to write and read `gtGetOp1`) and that's something developers do more often than searching for uses of a particular definitions. And it's not unusual for this weird practice to fail to work well at solving exactly the problem is supposed to solve - finding uses. Because everything `GenTree` related is `gt` prefixed you're out of luck if you're trying to find `GenTreeLclFld::gtFieldSeq` uses and not `GenTreeIcon::gtFieldSeq` uses.

There's a perfectly good way to find all uses of a function or data member - just rename it and compile. If the name happens to be bad and you're doing tons of changes around that function/data member then it's even better - rename it for good and include it in the PR so that reviewers can easily see what parts of the JIT code are affected by the change.

The only case where prefixes have some appeal is the `Compiler` class with its zillion members. Of course, it shouldn't be like that but it's a ton of work to split it so in the meantime using C style prefixes is a reasonable workaround to preserve one's sanity when dealing with `Compiler`. Sadly, even this is messed up in the JIT as lots of functions use the `fg` prefix even then more specific prefixes (`ssa`, `vn` etc.) would have made more sense.

Reply:
I'm not a fan of prefixes either and I understand your position.
However, I think it would be better to create a new issue from this comment (design discussion?). I think the main question is how to do such change (one massive PR that requires days of searching and renaming? do it incrementally somehow?), its cost and its benefits.

I do not think that it is productive not to use prefixes for the new methods/classes and keep them for the old.